### PR TITLE
Restructures OS X template as a bundled structure

### PR DESCRIPTION
**Pull request by martinjlowm**
- Updates the template for OS X to mimic the official structure of an OS
  X application.
- Supplies an ICNS file for icons of multiple sizes (16x16, 32x32,
  128x128, 256x256, 512x512 given in pixels)
- Corrects a typo in `osx/README.txt'

### CODE REVIEW

**Comment by mvdan** [comment]
@martinjlowm looks like you branched from an older master and there are conflicts to resolve. Could you rebase, please?

**Comment by martinjlowm** [comment]
Yes, of course!

File: osx/Contents/MacOS/Workcraft [hunk 1]
```diff
@@ -5,6 +5,8 @@ CURRENT_DIR="$PWD"
 
 # Change to Workcraft home directory and put it into the WORKCRAFT_HOME variable
 cd "$(dirname "$0")"
+cd ../Resources
```
Comment (mvdan) [inline osx/Contents/MacOS/Workcraft]: Join the two `cd` commands.

**Comment by mvdan** [comment]
Nothing else looks obviously wrong, but I have no idea about mac stuff. Perhaps @danilovesky or @jrbeaumont can chime in.

**Comment by martinjlowm** [comment]
It seems I missed the VXM tool tree after the rebase. It needs to be moved into `Contents/Resources/tools`.

**Comment by danilovesky** [comment]
Let us postpone it till tomorrow, cannot think clearly now.
And maybe @mvdan knows how to fix it in a clean way.

**Comment by mvdan** [comment]
It's fine to move the remaining files in a separate commit. Commits that only rename files do not noticeably add up to the size of a repo. (If you were thinking of rewriting the pushed history).

**Comment by martinjlowm** [comment]
Since the request is closed, new activity on my branch is ignored. I have committed the changes however (9f4d5f1).

**Comment by mvdan** [comment]
@martinjlowm feel free to open a new PR for those and we can merge them. Updating a merged PR is not something we should do, as undoing pushes to master is not a good thing at all.

**Pull request merged**